src/runtime/runtime.ts

```diff
@@ -592,13 +598,16 @@ export class ConfiguredRuntime implements Deps, SubscriptionsInterface {
     clientOptions?: {
       lang?: string;
       theme?: ClientTheme;
-    }
+    },
+    creationTimestamp?: number
```

Comment:
```suggestion
    private readonly creationTimestamp_ = 0
```

This change should remove the need for lines 609 and 563